_posts/2021-12-17-glean-in-2021.md

```diff
@@ -0,0 +1,188 @@
+---
+permalink: "/{{ year }}/{{ month }}/{{ day }}/glean-in-2021"
+title: "This Week in Glean: Looking back at Glean in 2021"
+published_date: "2021-12-17 15:00:00 +0100"
+layout: post.liquid
+data:
+  route: blog
+---
+
+(“This Week in Glean” is a series of blog posts that the Glean Team at Mozilla is using to try to communicate better about our work. They could be release notes, documentation, hopes, dreams, or whatever: so long as it is inspired by Glean.)
+All "This Week in Glean" blog posts are listed in the [TWiG index](https://mozilla.github.io/glean/book/appendix/twig.html)
+(and on the [Mozilla Data blog](https://blog.mozilla.org/data/category/glean/)).
+This article is [cross-posted on the Mozilla Data blog][datablog].
+
+[datablog]: https://blog.mozilla.org/data/2021/12/17/this-week-in-glean-glean-in-2021
+
+---
+
+A year ago I posted [Glean in 2021](/2020/12/18/glean-in-2021/) as a way to look into the future
+and set out a vision and plan for the project.
+Today I'm looking back at 2021, if we were able to follow up on my plans back then and look at all the other things we did for Glean.
+
+Let's start easy:
+According to [the index](https://mozilla.github.io/glean/book/appendix/twig.html) we wrote 21 This Week in Glean blog posts (including this one).
+Close enough to one every other week.
+Communicating about our work is important and TWiGs are one way to put our ideas and thoughts about the project out there.
+
+Let's first look at the topics I identified as important in last year's blog post.
+
+## The vision
+
+> The Glean SDK is a fully self-servable telemetry SDK, usable across different platforms.
+> It enables product owners and engineers to instrument their products and rely on their data collection,
+> while following Mozilla policies & privacy standards.
+
+I'd say we're in pretty good shape here.
+The docs on [how to add Glean to a project][adding-glean] need some updates,
+but we also added a post for a slightly different audience: [Integrating Glean for product managers][glean-for-pms],
+so it's not only for engineers anymore.
+We asssisted others when they integrated Glean, but more for general data and instrumentation knowledge.
+
+
+[adding-glean]: https://mozilla.github.io/glean/book/user/adding-glean-to-your-project/index.html
+[glean-for-pms]: https://mozilla.github.io/glean/book/user/integrating-glean-for-product-managers.html
+
+## The ideas
+
+### Metric types
+
+We specified & implemented 3 new metric types this year.
+The [`rate` metric][rate] has been implemented in the Rust SDK only and is available on Firefox Desktop.
+The [`url` metric][url] is available in all SDKs, but not yet exposed on Firefox Desktop.
+The [`text` metric][text] is currently JavaScript-only.
+
+However the `url` and `text` metric were not usable until recently because of bugs in our schema generation tool.
+We had to apply some sort-of hacky fixes to get them to work, but the pipeline now correctly ingests data for these metrics
```

Comment:
The way this is put, it does not make us look too trustworthy, specifically "We had to apply some sort-of hacky fixes to get them to work". Not really clear to me on how to rephrase, just a thought.